CHANGELOG.md

```diff
@@ -5,6 +5,28 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 17.03.2-ce (2017-05-29)
+
+### Networking
+
+- Fix a concurrency issue preventing network creation [#33273](https://github.com/moby/moby/pull/33273)
+
+### Runtime
+
+- Relabel secrets path to avoid a Permission Denied on selinux enabled systems [#33236](https://github.com/moby/moby/pull/33236)
```

Comment:
If the cherrypick PR didn't cherrypick all in one, I would prefer the cherrypick PR since it's closest to what actually is changed in the release branch. But in this case it bundles too much and it's hard to follow.